### ISSUE TITLE
Lab compass not working

### ISSUE THREAD

**Issue by loveuseless** [yznpku/LabCompass#72]
it does show that map is loaded before entering the lab
D
![image](https://user-images.githubusercontent.com/53101813/138752486-7748c88d-e720-411f-b621-d0f8f1395bd5.png)

but in the lab it shows nothing.
![image](https://user-images.githubusercontent.com/53101813/138752629-df8e7c10-3afa-4921-bce7-820f98bb44b2.png)

Had this for a fey patches now, please fix

**Comment by RoOoBerT** [comment]
Hi guys, I had the same issue occasionally for previous leagues.
Since Scourge league it did not work even once for the moment for me.

PS : Thanks for that awesome tools ! <3

**Comment by redrios** [comment]
hello! YES! I've been having this same issue for at least 5 leagues; up until now I managed to "restart" the app by erasing all the registry entries with labcompass name on them through regedit's find function. If your labcompass worked at some point in the past, it can regain usability

**Comment by Alistair1231** [comment]
https://github.com/yznpku/LabCompass/issues/52#issuecomment-402280787
this fixed it for me as well. 
tl;dr delete your `Documents\My Games\Path of Exile`  folder and restart PoE and LabCompass.
You may want to backup your filters first.

**Comment by Hen664** [comment]
My compass loads but does show direction non steam version, was wondering if it doesnt run on windows 11 if so can i get it to work, this is my location.
![Screenshot 2023-10-24 165705](https://github.com/yznpku/LabCompass/assets/102096649/cef40536-8166-48cb-8491-69aa4d74408d)
